### PR TITLE
Update MessageWrapper to support non-portrait orientations

### PR DESCRIPTION
**Pull request by cshfang**
Update MessageWrapper to support non-portrait orientations

#### Description of changes

Added support for all orientations for underlying `Modal` component. Available enums can be found [here](https://reactnative.dev/docs/modal#supportedorientations-ios)

#### Description of how you validated changes
Tested locally against sample app

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by calebpollman — APPROVED** [review]
LGTM, thanks @cshfang

File: packages/aws-amplify-react-native/src/InAppMessaging/components/MessageWrapper/MessageWrapper.tsx [hunk 1]
```diff
@@ -19,7 +19,11 @@ import { MessageWrapperProps } from './types';
 
 export default function MessageWrapper({ children, style }: MessageWrapperProps) {
 	return (
-		<Modal transparent visible>
+		<Modal
+			transparent
+			visible
+			supportedOrientations={['portrait', 'portrait-upside-down', 'landscape', 'landscape-left', 'landscape-right']}
```
Comment (Samaritan1011001) [inline packages/aws-amplify-react-native/src/InAppMessaging/components/MessageWrapper/MessageWrapper.tsx:25]: Question: this prop seems to be just iOS. Do you know if it's the same issue on Android? 
https://reactnative.dev/docs/modal#supportedorientations-ios

**Review by Samaritan1011001 — APPROVED** [review]
One question. but LGTM 👍  Thank you!

File: packages/aws-amplify-react-native/src/InAppMessaging/components/MessageWrapper/MessageWrapper.tsx (hunk 1 above)
Comment (cshfang) [reply packages/aws-amplify-react-native/src/InAppMessaging/components/MessageWrapper/MessageWrapper.tsx:25]: Good question. I'll have to double check. Would assume that on Android there's just no locking down modal orientations via RN

**Comment by cshfang** [comment]
Closing this in favor of larger overall carousel support PR

**Comment by cshfang** [comment]
Closing this in favor of larger overall carousel support PR

**Pull request closed without merging**